plugins/inputs/win_perf_counters/pdh.go

```diff
@@ -544,20 +545,20 @@ func PdhExpandWildCardPath(szWildCardPath string, mszExpandedPathList *uint16, p
 	ptxt, _ := syscall.UTF16PtrFromString(szWildCardPath)
 	flags := uint32(0) // expand instances and counters
 	ret, _, _ := pdh_ExpandWildCardPathW.Call(
-		uintptr(unsafe.Pointer(nil)), // search counters on local computer
-		uintptr(unsafe.Pointer(ptxt)),
-		uintptr(unsafe.Pointer(mszExpandedPathList)),
-		uintptr(unsafe.Pointer(pcchPathListLength)),
-		uintptr(unsafe.Pointer(&flags)))
+		// search counters on local computer
+		0,
```

Comment:
Maybe just restore the comment as well..
```suggestion
		0, // search counters on local computer
```